src/python/pants/backend/python/rules/pex.py

```diff
@@ -314,18 +317,17 @@ def log(self, *args, **kwargs) -> None:
             )
         )
 
-    sources_digest = (
-        request.input_files_digest if request.input_files_digest else EMPTY_DIRECTORY_DIGEST
```

Comment:
Was there any reason this wasn't just `request.input_files_digest or EMPTY_DIRECTORY_DIGEST`? That is a common and succinct idiom.

Reply:
Not that I can tell. I think it was an oversight.